contributors/devel/pull-requests.md

```diff
@@ -77,35 +106,240 @@ $ $RELNOTES -man # for details on how to use the tool
+
+Or, it could be related to best practices. One common issue is that the PR is too big to review. Let's say you've touched 39 files and have 8657 insertions. When your would-be reviewers pull up the diffs, they run away - this PR is going to take 4 hours to review and they don't have 4 hours right now. They'll get to it later, just as soon as they have more free time (ha!).
+
+We've got a detailed rundown of best practices, including how to avoid too-lengthy PRs, in the next section.
+
+But, if you've already followed the best practices and you still aren't getting any PR love, here are some
+things you can do to move the process along:
+
+   * Make sure that your PR has an assigned reviewer (assignee in GitHub). If not, reply to the PR comment stream asking for a reviewer to be assigned.
+
+   * Ping the assignee (@username) on the PR comment stream, and ask for an estimate of when they can get to the review.
+
+   * Ping the assignee by email (many of us have publicly available email addresses, or are the same as our GitHub handle @google.com or @redhat.com).
+
+   * Ping the [team](https://github.com/orgs/kubernetes/teams) (via @team-name) that works in the area you're submitting code.
+
+   * If you have fixed all the issues from a review, and you haven't heard back, you should ping the assignee on the comment stream with a "please take another look" (`PTAL`) or similar comment indicating that you are ready for another review.
+
+Read on to learn more about how to get faster reviews by following best practices.
+
+# Best Practices for Faster Reviews
+
+Most of this section is not specific to Kubernetes, but it's good to keep these best practices in mind when you're making a PR.
+
+You've just had a brilliant idea on how to make Kubernetes better. Let's call that idea Feature-X. Feature-X is not even that complicated. You have a pretty good idea of how to implement it. You jump in and implement it, fixing a bunch of stuff along the way. You send your PR - this is awesome! And it sits. And sits. A week goes by and nobody reviews it. Finally someone offers a few comments, which you fix up and wait for more review. And you wait. Another week or two go by. This is horrible.
+
+What went wrong?
+
+Let's talk about best practices so your PR gets reviewed quickly.
+
+## 0. Familiarize yourself with project conventions
+
+* [Development guide](development.md)
+* [Coding conventions](coding-conventions.md)
+* [API conventions](api-conventions.md)
+* [Kubectl conventions](kubectl-conventions.md)
+
+## 1. Is the feature wanted? Make a Design Doc or Sketch PR
+
+Are you sure Feature-X is something the Kubernetes team wants or will accept? Is it implemented to fit with other changes in flight? Are you willing to bet a few days or weeks of work on it?
+
+It's better to get confirmation beforehand. There are two ways to do this:
+
+- Make a proposal doc (in docs/proposals; for example [the QoS proposal](http://prs.k8s.io/11713))
```

Comment:
- Make a proposal doc (in docs/proposals; for example [the QoS proposal](http://prs.k8s.io/11713)), or reach out to the affected special interest group (SIG). Here's a [list of SIGs](https://github.com/kubernetes/community/blob/master/sig-list.md).